### ISSUE TITLE
Installing the GeoIP Database -> ERROR

### ISSUE THREAD

**Issue by DarkTexas** [Spuds/SMF-GeoIP#4]
The Database is not installed
Last Update Status: Database Import Error

**Comment by Spuds** [comment]
Yup, pretty common.  Its Not a mod issue, or an issue that the mod can overcome.  As stated in the readme:

If you do not have the zip module installed (the mod will inform you of this) then you may run into an out of memory issue when unzipping the large database. Without the zip module the memory requirements are 272M which may be a problem on some systems. If you get an out of memory error or white screen your options are to use the country only database or to use the manual install option which requires that you upload the unzipped CSV files and then mod will install them.

Most likely you have a memory limit, or a mysql import limit or the bulk import is disabled on your host.  Your only alternative (assuming you are not on a VPS) is to do the manual install and hope that works.

**Comment by DarkTexas** [comment]
Got an Windows Server, so.. :<
I'm kinda lost; but that mod is like necessary on our Gaming-Clan-Forums. :(

**Comment by Spuds** [comment]
Is it your own server?  Do you have root or command window access?

The basic problem is the size of the files from maxmind.  They work find when doing things from the command line, but once you try to install it via PHP you run in to all sorts of front end limitations.  When I ported this over to ElkArte I dropped the option to install the db this way since it was so problematic and instead opted to have the admin install geoIP capability on the server (but again you are dependent on the server)

**Comment by DarkTexas** [comment]
It's my Own Server; Yes, Full Access.
Maybe you can help me doing it in consideration of a small donation.
Please Contact me at c53440@yahoo.de.
